sqitch.plan

```diff
@@ -9,3 +9,4 @@ materialized_view_report [table_ghgr_import] 2019-04-25T21:07:39Z Dylan Leard <d
 materialized_view_organisation [table_ghgr_import] 2019-04-29T20:14:13Z Dylan Leard <dylan@button.is> # add materialized view organisation to schema ggrics_swrs
 materialized_view_facility [materialized_view_report] 2019-04-29T22:24:35Z Dylan Leard <dylan@button.is> # add materialized view facility to schema ggrics_swrs
 materialized_view_flat [materialized_view_report] 2019-04-30T18:11:08Z Dylan Leard <dylan@button.is> # add materialized view flat to schema ggircs_swrs
+materialized_view_identifier [materialized_view_facility] 2019-04-30T23:33:13Z Dylan Leard <dylan@button.is> # add materialized view identifier to schema ggircs_swrs
```

Comment:
wherever possible, it would be great to refactor migrated views to depend only on `ghgr_import`